source/reference/metrics.txt

```diff
@@ -0,0 +1,270 @@
+.. _app-services-metrics:
+
+==============================
+App Services Metrics Reference
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+App Services provides a variety of metrics related to App Services and Device
+Sync. You can access your metrics data with the Admin API's
+:admin-api-endpoint:`getMetrics <tag/metrics/operation/adminGetMetrics>` endpoint.
+
+The content on this page is provided for your information. The exact metrics
+returned may be subject to change.
+
+.. Source derived from https://docs.google.com/spreadsheets/d/1cjQopjPUsEKde6TlqO38B0Ko6n4vTwl4tgJ-LHSbf7Y/edit#gid=1354315462
+
+Units
+-----
+
+App Services expresses each metric in one of the following units:
+
+- BYTES_PER_SECOND
+- MILLISECONDS
+- SCALAR
+- SCALAR_PER_SECOND
+
+A **scalar** is a singular value representing a magnitude or count. Its exact
+interpretation depends on the metric.
+
+A **scalar per second** represents a count over time ("rate"). The value is
+effectively the total count in the requested time window divided by the number
+of seconds in the window.
+
+Some scalar per second values are expressed as a 95th percentile. This means
+that the reading for the given metric is below the given value 95% of the time
+and above it the other 5% of the time.
+
+Available Metrics
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 20 30
+
+  * - Metric Name
+    - Unit
+    - Description
+
+  * - ACTIVE_OPEN_SYNC_SESSIONS
+    - SCALAR
+    - Active open sync sessions.
+
+  * - AUTH_EGRESS_BYTES
+    - BYTES_PER_SECOND
+    - The amount of data transferred from the server due to
+      authentication requests.
+
+  * - AUTH_FAILED_REQUESTS
+    - SCALAR_PER_SECOND
+    -  failed authentication requests.
+
+  * - AUTH_RESPONSE_MS
+    - MILLISECONDS
+    - 95th percentile authentication response time.
+
+  * - AUTH_SUCCESSFUL_LOGIN
+    - SCALAR_PER_SECOND
+    - Rate of successful logins.
+
+  * - AUTH_SUCCESSFUL_REQUESTS
+    - SCALAR_PER_SECOND
+    - Rate of successful authentication requests.
+
+  * - AUTH_TOTAL_USERS
+    - SCALAR
+    - The total number of users.
```

Comment:
how does this work with the time span in the query params? 

i/e number of users at beginning of time span could be different from the number at the end